### PR TITLE
remove the unused verifyRemainingObjects function

### PR DESCRIPTION
**Pull request by tianshapjq**
**What this PR does / why we need it**:
erase useless function when reading code

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
none
```

### CODE REVIEW

File: test/integration/replicaset/replicaset_test.go [hunk 1]
```diff
@@ -112,32 +112,6 @@ func newMatchingPod(podName, namespace string) *v1.Pod {
 	}
 }
 
-// verifyRemainingObjects verifies if the number of the remaining replica
-// sets and pods are rsNum and podNum. It returns error if the
-// communication with the API server fails.
-func verifyRemainingObjects(t *testing.T, clientSet clientset.Interface, namespace string, rsNum, podNum int) (bool, error) {
```
Comment (nikhita) [inline test/integration/replicaset/replicaset_test.go]: @tianshapjq thanks for cleaning this up!

can you remove for the replication controller as well?

https://github.com/kubernetes/kubernetes/blob/0fcc38425843d973538ce743a907e4c98095f8a0/test/integration/replicationcontroller/replicationcontroller_test.go#L111

Comment (tianshapjq) [reply test/integration/replicaset/replicaset_test.go]: @nikhita sure thing~

**Comment by tianshapjq** [comment]
@nikhita done

**Comment by nikhita** [comment]
@tianshapjq can you update the commit message to `remove the unused verifyRemainingObjects function`? thanks!

**Comment by tianshapjq** [comment]
@nikhita done~

**Pull request merged**